metabase/metabase
# Show only saved questions and models when selecting a card for a parameter in a dashboard

Epic https://github.com/metabase/metabase/issues/26898

How to verify:
- Create a dashboard with a card and a `text -> dropdown` parameter mapped to a column in the card
- Edit dashboard -> Select parameter -> Click `Values from a model or question`
- If there are no models available, it should automatically expand to saved questions
- Make sure it is possible to save changes and open this modal again with the selected card

<img width="1123" alt="Screenshot 2023-01-04 at 17 06 35" src="https://user-images.githubusercontent.com/8542534/210585098-71572bce-bc71-4c6b-af0a-cdc01a54bf49.png">
